pkgs/by-name/ge/gemini-cli-nightly/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  writeShellApplication,
+  cacert,
+  curl,
+  gnused,
+  jq,
+  nix-prefetch-github,
+  prefetch-npm-deps,
+  gitUpdater,
+  ...
+}:
+
+buildNpmPackage (finalAttrs: {
+  pname = "gemini-cli";
+  version = "0.1.9-nightly.250708.137ffec3";
+
+  src = fetchFromGitHub {
+    owner = "google-gemini";
+    repo = "gemini-cli";
+    rev = "821abfc456cb4664f8e7cfdc5856b1dd6564fecd";
+    hash = "sha256-2w28N6Fhm6k3wdTYtKH4uLPBIOdELd/aRFDs8UMWMmU=";
+  };
+
+  npmDeps = fetchNpmDeps {
+    inherit (finalAttrs) src;
+    hash = "sha256-yoUAOo8OwUWG0gyI5AdwfRFzSZvSCd3HYzzpJRvdbiM=";
+  };
+
+  postPatch = ''
+    sed -i 's/"version": "0.1.5"/"version": "${finalAttrs.version}"/' package.json
+    sed -i 's/"version": "0.1.5"/"version": "${finalAttrs.version}"/' packages/cli/package.json
+    sed -i 's/"version": "0.1.5"/"version": "${finalAttrs.version}"/' packages/core/package.json
```

You can use a `for` loop
```suggestion
    for file in {,packages/cli,packages/core}package.json; do
      sed -i 's/"version": "0.1.5"/"version": "${finalAttrs.version}"/' $file
    done
```
or with pure Bash functions (preferred):
```suggestion
    for file in {,packages/cli,packages/core}package.json; do
      substituteInPlace $file --replace-fail '"version": "0.1.5"' "version": "${finalAttrs.version}"'
    done
```